packages/renderer/src/lib/recommendation/ExtensionBanner.svelte

```
@@ -39,11 +39,11 @@ async function onClose(): Promise<void> {
       buttons: ['Keep', 'Hide'],
     });
 
-    if (result?.response === 1) {
+    if (result?.response === 'Hide') {
       await window.updateConfigurationValue(`extensions.ignoreBannerRecommendations`, true, 'DEFAULT');
```

Biome (2.5.0)

[error] Do not use template literals if interpolation and special-character handling are not needed.

(lint/style/noUnusedTemplateLiteral)